test/fixtures/tls-connect.js

```diff
@@ -0,0 +1,100 @@
+// One shot call to connect a TLS client and server based on options to
+// tls.createServer() and tls.connect(), so assertions can be made on both ends
+// of the connection.
+
+const common = require('../common');
+const fs = require('fs');
```

Comment:
reorganized as above, except that I always sort require statements lexicographically, I'm a bit surprised the test guide doesn't.